lynx/src/main/java/com/github/pedrovgs/lynx/LynxConfig.java

```diff
@@ -77,8 +91,12 @@ public String getFilter() {
     return filter;
   }
 
+  public TraceLevel getFilterTraceLevel() {
+    return filterTraceLevel;
+  }
+
   public boolean hasFilter() {
-    return filter != null;
+    return filter != null && !"".equals(filter) || !TraceLevel.ALL.equals(filterTraceLevel);
```

Comment:
You have changed the default filter value to ``""`` and added a guard to avoid a null filter, now you don't need this ``filter != null``. You can change this if to:

``` java

public boolean hasFilter(){
    return !"".equals(filter) || !TraceLevel.VERBOSE.equals(filterTraceLevel);
}

```